contributors/design-proposals/ha_master.md

```diff
@@ -183,10 +183,10 @@ A pseudo-code for adding a new master replica using managed DNS and a loadbalanc
   4. Update DNS to point to the load balancer.
 2. Clone existing master (create a new VM with the same configuration) including
    all env variables (certificates, IP ranges etc), with the exception of
-   `INITIAL_ETCD_CLUSTER`.
+   INITIAL_ETCD_CLUSTER.
 3. SSH to an existing master and run the following command to extend etcd cluster
    with the new instance:
-   `curl <existing_master>:4001/v2/members -XPOST -H "Content-Type: application/json" -d '{"peerURLs":["http://<new_master>:2380"]}'`
+   curl <existing_master>:4001/v2/members -XPOST -H "Content-Type: application/json" -d '{"peerURLs":["http://<new_master>:2380"]}'
```

Comment:
` were needed there, it makes gfm to render them correctly like:  `curl <existing_master>:4001/v2/members -XPOST -H "Content-Type: application/json" -d '{"peerURLs":["http://<new_master>:2380"]}'`